src/util/versionstore.cpp

```diff
@@ -19,32 +18,54 @@
 #undef WIN32
 #endif
 
-#include <lame/lame.h>
 #include <FLAC/format.h>
 #include <chromaprint.h>
+#include <lame/lame.h>
+#include <portaudio.h>
 #include <rubberband/RubberBandStretcher.h>
-#include <taglib/taglib.h>
 #include <sndfile.h>
+#include <taglib/taglib.h>
 #include <vorbis/codec.h>
-#include <portaudio.h>
 
-#include "defs_version.h"
-#include "build.h" // Generated by SCons
+#define VERSION_STORE
+#include "version.h"
+
+namespace {
+
+const QString kMixxxVersion = QStringLiteral(MIXXX_VERSION);
+const QString kMixxx = QStringLiteral("Mixxx");
+#ifdef BUILD_BRANCH
```

Comment:
Unnecessary, this is always defined

Reply:
if this is unintended, you need to use `cmakedefine` instead of `define`. Culprit: The name and the variable need to match. But I think `GIT_BRANCH` is probably more descriptive anyway.